rule_schema_v1.yaml

```diff
@@ -644,6 +644,10 @@ properties:
         dest-language:
           title: Language to process the extracted result of this rule as
           type: string
+        # EXPERIMENTAL
+        json:
```

Comment:
I think a transform: with an enum with 3 choices,
concat_json_string_array | unquote_string
would be clearer and support further improvements.

Reply:
Done.